astropy/table/table.py

```diff
@@ -2064,6 +2065,51 @@ def rename_column(self, name, new_name):
 
         self.columns[name].info.name = new_name
 
+    def rename_columns(self, names, new_names):
+        ''' ****
+        Renames multiple columns.
+
```

Comment:
You could include a `.. versionadded: 2.0` here (leave a blank line before and after).